workloads/server.hbs.md

```diff
@@ -105,3 +105,285 @@ The following values are valid within the `ports` argument:
 | `port` | The port on which the application is exposed to the rest of the cluster |
 | `containerPort` | The port on which the application listens for requests. Defaults to `port` if not set. |
 | `name` | A human-readable name for the port. Defaults to `port` if not set. |
+
+
+## <a id="params"></a> Securing `server` workloads
```

Comment:
I would call this "Exposing `server` workloads outside the cluster".

Reply:
Also, you need a unique `id` so people can link to this.

Reply:
done